### PR TITLE
added What is Kafka primer to landing page and info to docs

### PR DESCRIPTION
**Pull request by jracusin**
addressing issue #57 What is Kafka

### CODE REVIEW

File: app/routes/index.mdx [hunk 1]
```diff
@@ -34,6 +34,14 @@ The General Coordinates Network is the modern evolution of the [Gamma-ray Coordi
...
 - [**Circulars**](/circulars) are human-readable, citable, rapid but generally not real-time, bulletins observations, quantitative near-term predictions, requests for follow-up observations, or future observing plans.
 
+## Kafka Streaming Platform
+
+[Apache Kafka](https://kafka.apache.org) is an open-source distributed event streaming platform used in both commerical web applications
+and in recent years, it has been widely adopted by the astronomical community. Utilizing a rich development environment with
+open-source and commericially available tools, GCN is running a cloud-based high-reliability Kafka broker that distributes
+GCN notices to users. Kafka is the platform of streaming data, and is agnostic of data format. However, GCN will transition to
+distributing native Kafka Avro records (see [roadmap](docs/roadmap)), which can be losslessly converted between text based JSON records and a binary format.
+
```
Comment (lpsinger) [inline app/routes/index.mdx:44]: This might make more sense in an FAQ section of the docs.

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/docs/index.md [hunk 2]
```diff
@@ -13,7 +13,7 @@ GCN is the established platform for publishing discoveries and follow-up of gamm
 
 ## Client Configuration
 
-GCN [Client Configuration](/docs/client) documentation for [**hop-client**](/docs/client#hop-client), [**adc-streaming**](/docs/client#adc-streaming) and [**confluent-kafka**](/docs/client#confluent-kafka).
+To begin receiving GCN notices via kafka, users have three options for setting up their listeners. GCN [Client Configuration](/docs/client) documentation for [**hop-client**](/docs/client#hop-client), [**adc-streaming**](/docs/client#adc-streaming) and [**confluent-kafka**](/docs/client#confluent-kafka). Once the listener is setup, users can specify which Kafka topics they wish to follow for different GCN notice types.
```
[the remark this answers is not in the document]
Comment (jracusin) [reply app/routes/docs/index.md]: I changed the wording to reflect what it says on the Client page, with two lightweight wrappers.

File: app/routes/index.mdx (hunk 1 above)
Comment (jracusin) [reply app/routes/index.mdx]: I made a FAQ page, and moved this there, but none of the react components work.

Comment (jracusin) [reply app/routes/index.mdx]: I removed accordion and search, and it's just a plain md page now.

File: app/routes/docs/faq.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+meta:
+  title: GCN - FAQ
+---
+
+# Frequently Asked Questions (FAQ)
```
Comment (lpsinger) [inline app/routes/docs/faq.md:6]:
```suggestion
# Frequently Asked Questions
```

File: app/routes/docs/index.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ GCN is the established platform for publishing discoveries and follow-up of gamm
 
 ## Client Configuration
 
-GCN [Client Configuration](/docs/client) documentation for [**hop-client**](/docs/client#hop-client), [**adc-streaming**](/docs/client#adc-streaming) and [**confluent-kafka**](/docs/client#confluent-kafka).
+To begin receiving GCN notices via kafka, users have two options for setting up their listeners. GCN [Client Configuration](/docs/client) documentation describes lightweight wrappers around [**kafka.js**](/docs/client#node.js) and [**confluent-kafka**](/docs/client#confluent-kafka). Once the listener is setup, users can specify which Kafka topics they wish to follow for different GCN notice types.
```
Comment (lpsinger) [inline app/routes/docs/index.md:16]:
```suggestion
To begin receiving GCN notices via Kafka, users have two options for setting up their listeners. The [Client Configuration](/docs/client) section describes how to connect to Kafka using [Python](/docs/client#python) or [JavaScript](/docs/client#nodejs). Once the listener is set up, users can specify which Kafka topics they wish to follow for different GCN notice types.
```

File: app/routes/docs/faq.md [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+
+## What is Kafka?
+
+[Apache Kafka](https://kafka.apache.org) is an open-source distributed event streaming platform used in both commerical web applications
+and in recent years, it has been widely adopted by the astronomical community. Utilizing a rich development environment with
+open-source and commericially available tools, GCN is running a cloud-based high-reliability Kafka broker that distributes
+GCN notices to users. Kafka is the platform of streaming data, and is agnostic of data format. However, GCN will transition to
+distributing native Kafka Avro records (see [roadmap](docs/roadmap)), which can be losslessly converted between text based JSON records and a binary format.
```
Comment (lpsinger) [inline app/routes/docs/faq.md:14]:
```suggestion
GCN notices to users. Kafka transports the alerts, but it is agnostic about the data format.
```

**Review by lpsinger — CHANGES_REQUESTED** [review]

File: app/routes/docs/faq.md (hunk 5 above)
Comment (jracusin) [reply app/routes/docs/faq.md:14]: We reference JSON on the "one format, one protocol" image, so I was trying to provide context for that.

**Review by lpsinger — APPROVED** [review]

**Pull request merged**